### ISSUE TITLE
Cannot install from chrome webstore

### ISSUE THREAD

**Issue by asm20** [NeverDecaf/chromium-web-store#13]
Trying to install a plugin from the chrome webstore (ublock Origin for example) yields the message "Apps, extensions, and user scripts cannot be added from this website". Currently using Ungoogled Chromium version 80.0.3987.132 running on Windows 10

**Comment by Rowisi** [comment]
I suspect you have tried to install uBlock from the direct download link instead of going to chrome extensions website.

**Comment by asm20** [comment]
If by direct link you mean the “Add to Chromium” button on the Chrome Web Store, then yes that is what I am doing. What is the correct way to do this then?

**Comment by Rowisi** [comment]
I thought you tried to install it with direct link like this:
`https://clients2.google.com/service/update2/crx?response=redirect&acceptformat=crx2,crx3&prodversion=[VERSION]&x=id%3D[EXTENSION_ID]%26installsource%3Dondemand%26uc` 

That warning was triggered when I tried that but everything works fine when I try to install extensions from chrome store website.

**Comment by asm20** [comment]
Yeah I have not tried using a direct link. To clarify, this warning appears when trying to install from the Chrome Web Store.

**Comment by NeverDecaf** [comment]
I have the same version of ungoogled chromium (80.0.3987.132, from macchrome repo) and am not experiencing this issue. Did you set the flag `chrome://flags/#extension-mime-request-handling` to `Always prompt for install`?

**Comment by asm20** [comment]
Hello there. Because of various other issues I switched from the 64-bit to the 32-bit build of Ungoogled Chromium. I have set the extension-mime-requirest-handling flag and enabled developer mode in the extensions tab. The "Apps, extensions, and user scripts cannot be added from this website" no longer comes up, but I cannot seem to actually install anything. The install seems to stall at the "Adding to Chromium" phase and it never completes. I am not sure if this is an issue with this plugin or not. If it isn't then I think this issue can be safely closed.
